letsencrypt/cli.py

```diff
@@ -163,7 +166,34 @@ def _init_le_client(args, config, authenticator, installer):
     return client.Client(config, acc, authenticator, installer, acme=acme)
 
 
-def run(args, config, plugins):
+def _find_duplicative_certs(domains, config, renew_config):
+    """Find existing certs that duplicate the request."""
+
+    identical_names_cert, subset_names_cert = None, None
+
+    configs_dir = renew_config.renewal_configs_dir
+    cli_config = configuration.RenewerConfiguration(config)
+    for renewal_file in os.listdir(configs_dir):
+        full_path = os.path.join(configs_dir, renewal_file)
+        rc_config = configobj.ConfigObj(renew_config.renewer_config_file)
+        rc_config.merge(configobj.ConfigObj(full_path))
+        rc_config.filename = full_path
+
+        candidate_lineage = storage.RenewableCert(rc_config, config_opts=None,
```

Comment:
As mentioned before, this might throw exceptions. This is in particular important, because we are reading user input, a renewal configuration file that might be broken. Please at least add a `TODO` comment inline, so we fix before launch. Alternatively you could fix it here :) How about:

```python
try:
    candidate_lineage = storage.Renewa...
except SomeException:
    logger.warning("Renewal configuration file %s is broken. Skipping.", full_path)
    continue
```